app/src/main/java/io/token/sample/Application.java

```diff
@@ -43,12 +43,17 @@ public static void main(String[] args) throws IOException {
 
         // Log in using the test merchant account created
         // by the setup utility:
+        // Comment(Colin): rename to tokenMember?
         Member loggedInMember = tokenIO.login("m:28i4rgDUaM7sUBMCaP4ZfHJpnmMX:5zKtXEAq");
 
         // Initializes the server
         Spark.port(3000);
 
-        // Endpoint for transfering, called by client side after user approval
+        // Endpoint for transfer payment, called by client side after user approval
+
+        // Comment(Colin): We might want to consider adding a comment about what happens if the flow
+        // is broken (token is created and endorsed but the transfer endpoint here isn't called).
```

Comment:
Ouch, if we miss the transfer we thus also miss the shipping info. Adding a comment, but I'm backing away from the "you should try to hunt down the token and redeem it" thinking.